src/python/director/teleoppanel.py

```diff
@@ -1057,7 +1057,7 @@ def __init__(self, panel, jointGroups=None):
 
         if jointGroups is None:
             # Add only these joint groups:
-            telopJointGroupNames = ['Back', 'Base', 'Left Arm', 'Right Arm', 'Neck']
+            telopJointGroupNames = ['Back', 'Base', 'Left Arm', 'Right Arm', 'Neck', "Left Leg", "Right Leg"]
```

Comment:
this adds the left and right legs to this widget. I'd prefer not to add these but sometimes they can be useful